documentation/CAMARA-Security-Interoperability.md

```diff
@@ -203,7 +203,7 @@ This document defines that the authentication server SHOULD not return an id tok
 This document defines the following error handling for a missing "openid" value in scope. 
 Please refer to [Authentication Error Response](https://openid.net/specs/openid-connect-core-1_0.html#AuthError).
 
-If "openid" is missing in the scope value but a claim that is [standardized in OIDC](https://openid.net/specs/openid-connect-core-1_0.html#Claims) is requested, then the Authorization Server returns an HTTP response code of 400 (Bad Request) and an error invalid_request.
+If "openid" is missing in the scope value but a claim that is [standardized in OIDC](https://openid.net/specs/openid-connect-core-1_0.html#Claims) is requested, then the Authorization Server returns an error `invalid_request`.
```

Comment:
@AxelNennker I understand that you are changing the phrase for a more generic one that only refers to `invalid_request` error, but does not specify whether it should be an HTTP error or not, because in the case of auth code for example, it would be provided in the redirect URL. Right?

Reply:
In OIDC authorization code flow, as long as the redirect_uri, is valid the error is returned via the redirect_uri.
```
HTTP/1.1 302 Found
Location: https://client.example.org/cb?
    error=invalid_request
    &state=af0ifjsldkj
```